tests/integration/components/new-project-modal/component-test.ts

```diff
@@ -33,6 +34,6 @@ module('Integration | Component | new-project-modal', hooks => {
             closeModal=closeModal
         }}`);
         assert.dom('.modal').exists();
-        assert.dom('.modal-title').hasText('Create new project');
+        assert.dom('.modal-title').hasText('t:new_project.header:()');
```

Comment:
For component integration tests, you don't verify the end text, only the token that ember-intl adds to the page. If you want to verify the actual text is pulled from a translation document, use an acceptance test.

Reply:
Turns out we could have been doing something similar with ember-i18n as well, but didn't:

```ts
import { t } from 'ember-i18n/test-support';

assert.dom('.modal-title').hasText(t('new_project.header'));
```